.github/workflows/dev_deploy.yml

```diff
@@ -15,17 +15,29 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: '3.x'
-      - name: Run unit tests
-        run: docker-compose run web pytest --cov-config=.coveragerc --cov=ops_site --cov-report term-missing
```

Comment:
a temporary removal to pave the way for reusable workflows instead of more copy pasta